legacy/src/main/java/macrobase/analysis/summary/itemset/result/RiskRatioResult.java

```diff
@@ -0,0 +1,21 @@
+package macrobase.analysis.summary.itemset.result;
+
+public class RiskRatioResult {
+    private double rr = 0;
+    private double correction = 0;
+
+    public RiskRatioResult(double rr, double correction) {
+        this.rr = rr;
+        this.correction = correction;
+    }
+
+    public RiskRatioResult(double rr) {
+        this.rr = rr;
+    }
+
+    public double getRiskRatio() { return rr; }
```

Comment:
This can just be get() and then getCorrected() since we know this is a risk ratio result.